internal/node/node_launcher.sh

```diff
@@ -207,7 +206,7 @@ if [ "${EXPECTED_EXIT_CODE}" -eq "0" ]; then
   # handled by the node process.
   # If we had merely forked a child process here, we'd be responsible
   # for forwarding those OS interactions.
-  exec "${node}" "${NODE_OPTIONS[@]}" "${MAIN}" "${ARGS[@]}"
+  exec "${node}" "${ARGS[@]}" "${NODE_OPTIONS[@]}" "$(pwd)/TEMPLATED_script_path"
```

Comment:
Why the change from `MAIN` to `$(pwd)/TEMPLATED_script_path`? Does it not work with the relative path? If so why not change the value of `MAIN`?

Reply:
because `MAIN` is a node script that patches the resolver to fix the resolve order for node_modules for some cases (like Angular). Since we don't have node modules anymore I replaced that with just invoking the entry script directly.

We can change the value of MAIN though, that will also work